code-studio/src/css/plc/teacher_dashboard.scss

```diff
@@ -57,27 +59,53 @@
 
     .module_assignment {
       border-radius: 5px;
-      color: white !important;
       font-family: "Gotham 4r", sans-serif;
       text-decoration: none;
       margin: 10px;
       padding: 5px;
 
+      background-color: #0094CA;
+      color: white !important;
+
+      position: relative;
+
+      .ribbon-wrapper {
+        width: 90px;
+        height: 100%;
+        overflow: hidden;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+
+        .ribbon {
+          -webkit-transform: rotate(45deg);
+          -moz-transform:    rotate(45deg);
+          -ms-transform:     rotate(45deg);
+          -o-transform:      rotate(45deg);
+          position: relative;
+          padding: 7px 0;
+          left: -1px;
+          top: 10px;
+          width: 120px;
+        }
+      }
+
       .right_aligned {
         float: right;
       }
     }
 
     .not_started {
-      background-color: #0094ca;
+      background-color: $white;
+      border: 1px #bbb solid;
     }
 
     .in_progress {
-      background-color: #EFCD1C;
+      background-color: #EFCD1C !important;
```

Comment:
Could you add the new green and yellow colors to color.scss since we'll hopefully want to transition to using these? @breville if you want to input on name: I would name it $completed_green and $progress_yellow or something like that, listed under $goldenrod. (lol what color is that)